mackerel-plugin-aws-waf/main.go

```diff
@@ -0,0 +1,7 @@
+package main
+
+import "github.com/holidayworking/mackerel-plugin-aws-waf/lib"
```

Comment:
The import path is pointing to your repository, this would need to be fixed as well.